PRFCS/prfc-2.md

```diff
@@ -118,10 +122,10 @@ Returns `None` if spender is not specified.
 fn transfer(token_id: TokenID, to_address: Option<PublicAddress>)
 ```
 
-Transfers the token identified by `token_id` from the account identified by `txn.signer` (its owner), to the account identified by `to_address`. If `to_address` is None, the token will be burnt.
+Transfers the token identified by `token_id` from the account identified by its owner (`calling_account`), to the account identified by `to_address`. If `to_address` is None, the token will be burnt.
```

Comment:
Can we say: 
Transfers the token identified by `token_id` from the **_owner account identified by `calling_account`_**, to the account identified by `to_address`. If `to_address` is None, the token will be burnt.